src/exports.jl

```diff
@@ -149,14 +148,14 @@ export
     assemble!,
     finish_assemble,
 
-# VTK export
-    vtk_grid,
-    vtk_point_data,
-    vtk_cell_data,
-    vtk_nodeset,
-    vtk_cellset,
-    vtk_save,
-
+# export data (using e.g. Ferrite.VTKFile)
```

Comment:
Chose not to export `VTKFile` since `WriteVTK.jl` use that name internally as well (supertype of e.g. `DatasetFile`), to make it clear that this one belongs to Ferrite.